local-volume/utils/Makefile

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
Instead of using top level `utils` folder, this tool should probably be in a subfolder. This change will enable us to add other tools later.

Reply:
sounds good